TODO.md

```diff
@@ -0,0 +1,28 @@
+- ~~Update _React_ to v16.x~~
```

Comment:
Is there a better way to store TODOs? As issues, perhaps?

Reply:
Clarifying @RichardLitt's *slight hint* (😉), issues would be a great place for tracking the TODOs. That way others can contribute or keep track. Don't necessarily have to do one issue per task, could all be in one big "update all the things" issue, or broken down to smaller ones as you see fit.

Reply:
:) Maybe a TODO file is the best! I don't know! I just know that I tend to use issues, and that the rest of the Orbit community seems to. :)

Reply:
Yea issues are probably the correct way here :) The "todo" file began its life as simple file for me to keep track but now that it is becoming larger I will swap to using proper issues.